eval/eval.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+eval_store="$TMPDIR/eval"
+
+# ---------------------------------------------------------------------
+
+echo "::group::Nix Evaluation"
+
+ci_json="$(nix eval .#__std.ci."$(nix eval --raw --impure --expr 'builtins.currentSystem')" --json)"
+
+echo "$ci_json" >> "$GITHUB_OUTPUT"
```

Comment:
Ah! here it is! :smile:

Wouldn't this need a proper key like `eval-result::$ci_json` so that we can identify it and pass it on to the next actions?

Reply:
yep that needs to be fixed, although would it be a `=` and not a `::`? I'm not to clear on the difference but I've seen both.

Reply:
Absolutly:

https://github.com/input-output-hk/cardano-gov-world/blob/main/.github/workflows/ci.yml#L57